### ISSUE TITLE
1.13: Re-add Realtime

### ISSUE THREAD

**Issue by cakoyo** [Akarin-project/Akarin#43]
After Sponge update to 1.13.

**Comment by shinohara-rin** [comment]
what is _realtime_

**Comment by cakoyo** [comment]
@shinohara-rin realtime is a sponge feature, to make some tick based function tide to wallclock time, cause a better game experience in low tps situation.

**Comment by shinohara-rin** [comment]
I see, but I don't know how this _realtime_ feature would have anything to do with low TPS. will this save some tick loop?

**Comment by cakoyo** [comment]
No, for example when the TPS is 10, while everything needs double time to finish.
